scss/_utilities_vertical-spacing.scss

```diff
@@ -17,4 +17,28 @@
       }
     }
   }
+
+  @mixin vf-section-spacing-vertical($spacing) {
```

Comment:
Mixins can't be defined within other mixins, so this would need to be extracted to top level in this file (above `@mixin vf-u-vertical-spacing($start: -3, $end: 3)` I guess.

This is causing build and test errors. Locally it should fail too if you run `dotrun build` or `dotrun test`.